tests/python/rest_api/test_tasks.py

```diff
@@ -15,9 +18,17 @@
 
 from shared.utils.config import get_method, make_api_client, patch_method
 from shared.utils.helpers import generate_image_files
+from shared.utils.s3 import make_client
 
 from .utils import export_dataset
 
+sys.path.append(
+    os.path.join(
+        os.path.dirname(os.path.dirname(os.path.abspath(__file__))).split("tests")[0], "utils"
+    )
+)
+from dataset_manifest import ImageManifestManager
+
```

Comment:
I suggest to do one of the following:
- create a manifest via the server docker image as per documentation, the image must be installed anyway to run the tests
- create a separate package from the dataset_manifest, require it in the test deps (#5096)
- require dataset_manifest deps in the test deps and add it to the tests PATH

Reply:
The first option has been implemented